src/Loadbalancers/AclClient.php

```diff
@@ -44,7 +76,84 @@ public function getById($id)
     {
         $response = $this->request("GET", "v2/acls/$id");
         $body = $this->decodeJson($response->getBody()->getContents());
-        return $this->serializeAcl($body->data);
+        return new Acl($this->apiToFriendly($body->data, self::MAP));
+    }
+
+    public function getHeaders($id, $page = 1, $perPage = 15, $filters = [])
+    {
+        $filters = $this->friendlyToApi($filters, self::HEADER_MAP);
+        $page = $this->paginatedRequest("v2/acls/$id/headers", $page, $perPage, $filters);
+
+        $page->serializeWith(function ($item) {
+            return new Header($this->apiToFriendly($item, self::HEADER_MAP));
+        });
+        return $page;
+    }
+
+    public function getMatches($id, $page = 1, $perPage = 15, $filters = [])
+    {
+        $filters = $this->friendlyToApi($filters, self::MATCH_MAP);
+        $page = $this->paginatedRequest("v2/acls/$id/matches", $page, $perPage, $filters);
+
+        $page->serializeWith(function ($item) {
+            return new Match($this->apiToFriendly($item, self::HEADER_MAP));
```

Comment:
```suggestion
            return new Match($this->apiToFriendly($item, self::MATCH_MAP));
```
Using the wrong array of values